jonathaneeckhout/jdungeon
# chore(beehave): upgrade beehave from 2.7.5 to 2.7.6

Hopefully this fixes some of the issues with the debugger as it is quite nice to be able to visualize the behavior tree's now that they are getting a bit complex.

Testing Scenarios:
- Enemy AI should work the exact same is it did prior to upgrade.
- Verify Aggressive TreeGuys still work as intended. Aggro range, attacking, leashing. Wander/Idle otherwise.
- Verify Sheep Flee on combat and Wander.
